service/static/js-src/muk.institution.js

```diff
@@ -4547,6 +4547,7 @@ $.extend(true, edges, {
                 var orderId = edges.css_id(namespace, "order", this);
                 var toggleId = edges.css_id(namespace, "toggle", this);
                 var resultsId = edges.css_id(namespace, "results", this);
+                var countClass = edges.css_classes(namespace, "count", this);
                 var results = "Loading...";
                 if (ts.values !== false) {
```

Comment:
You should be able to streamline this by setting  `results = "No data available"` as an else {} to the following if statement:

`if (ts.values && ts.values.length > 0) { 
....
....
}
else {
  results = "No data available";
}`